include/dfrn.php

```diff
@@ -207,6 +216,11 @@ public static function feed($dfrn_id, $owner_nick, $last_update, $direction = 0,
 			dbesc($sort)
 		);
 
+		if (!dbm::is_result($r)) {
```

Comment:
The query may fail (e.g. database crashed/restarted), so better is to validate if all queries really run successful.